From 96182937a75447f3e80bbb76828f2ba342d0b392 Mon Sep 17 00:00:00 2001
From: Volodymyr Samotiy <volodymyrs@nvidia.com>
Date: Thu, 2 Oct 2025 16:23:46 +0300
Subject: [PATCH 2/2] Make system-health service starts after hw-management to
 avoid failures

On SN2410, it can fail to read the file led_status_capability if it starts from ONIE
---
 debian/hw-management.hw-management.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/hw-management.hw-management.service b/debian/hw-management.hw-management.service
index cd7fb7e2..e6f96743 100755
--- a/debian/hw-management.hw-management.service
+++ b/debian/hw-management.hw-management.service
@@ -3,7 +3,7 @@ Description=Chassis HW management service of Nvidia systems
 Documentation=man:hw-management.service(8)
 Wants=hw-management-sysfs-monitor.service
 Wants=hw-management-fast-sysfs-monitor.service
-Before=determine-reboot-cause.service
+Before=determine-reboot-cause.service system-health.service watchdog-control.service
 
 [Service]
 Type=oneshot
-- 
2.39.5

